src/routes.js

```diff
@@ -63,16 +64,4 @@ const getRoutes = (vms) => ([
   },
 ])
 
-/**
- * Return array of objects that describe vertical menu
- * @return {array}
- */
-const getMenu = () => ([
-  {
-    icon: 'pficon pficon-virtual-machine',
-    title: msg.virtualMachines(),
-    to: '/',
-  },
-])
-
-export { getRoutes, getMenu }
+export { getRoutes }
```

Comment:
I think it would enough to export getRoutes as default.

Reply:
Done.